docs/ref/models/querysets.txt

```diff
@@ -1887,6 +1887,45 @@ See the :doc:`/topics/db/sql` for more information.
   filtering. As such, it should generally be called from the ``Manager`` or
   from a fresh ``QuerySet`` instance.
 
+``comment()``
+~~~~~~~~~~~~~
+
+.. versionadded:: 4.2
+
+.. method:: comment(message)
+
+Adds an SQL comment to be inserted into the resultant SQL statement, after
+``SELECT``/``UPDATE``, and if applicable, ``DISTINCT``.. For example::
```

Comment:
I think we should support them for `DELETE` and `INSERT` as well.

Reply:
Haven't looked into how this could be implemented for `INSERT` but I agree, lets try and get support for those as well.! The special one is probably `DELETE`, would one except the comment to be added for the `SELECT` statements generated as well when running `delete()` or only the `DELETE`(s)? If we would want to piggyback optimizer hints on this we should probably only add comments for the actual `DELETE` statement in that case.